### ISSUE TITLE
game crashes when starting the crownfall act IV queen gambit encounter game on macos.

### ISSUE THREAD

**Issue by fanik05** [ValveSoftware/Dota2-Gameplay#22799]
### Description

My game just crashes when press play in the minigame of queen gambit encounter with an error message shown in the screen shot.

My system config:

- Macbook m1 pro 14 inch 2021
- 16GB RAM 
- 1 TB SSD
- macOS version 15.1

### Screenshots

<img width="1436" alt="Screenshot 2024-11-13 at 12 34 48 PM" src="https://github.com/user-attachments/assets/cf4a6fbc-65b6-4fa8-bda0-c41040b6afb8">

**Comment by buttersome** [comment]
Yeah a lot of these mini games would start crashing Dota if you play long enough, then exit the mini game and enter it again. Restarting steam is necessary to solve the crashing. 

But this Nest of Thrones encounter is unplayable, it just keeps crashing once you press start in that screenshot.

**Comment by Degoya-Studio** [comment]
same, please fix this Valve

**Comment by fakedol1** [comment]
> ### Description
> My game just crashes when press play in the minigame of queen gambit encounter with an error message shown in the screen shot.
> 
> My system config:
> 
> * Macbook m1 pro 14 inch 2021
> * 16GB RAM
> * 1 TB SSD
> * macOS version 15.1
> 
> ### Example Match ID (and possibly Timestamp)
> _No response_
> 
> ### Screenshots
> <img alt="Screenshot 2024-11-13 at 12 34 48 PM" width="1436" src="https://private-user-images.githubusercontent.com/48475054/385875976-cf4a6fbc-65b6-4fa8-bda0-c41040b6afb8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE2MTI0NTQsIm5iZiI6MTczMTYxMjE1NCwicGF0aCI6Ii80ODQ3NTA1NC8zODU4NzU5NzYtY2Y0YTZmYmMtNjViNi00ZmE4LWJkYTAtYzQxMDQwYjZhZmI4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTE0VDE5MjIzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWRiMGNkZmZkMjUwMzFlOTZjODQ0ZGI4NjliY2FhY2UxZGIxOTA2MWQyY2ZjMWU2MzNlY2MzNTIwZmY0OGQzNzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.FB1s-x0_yiSpgMOhum4uGi8ECcIASRcYf6ljEAaOX_U">

same issue, please do something about this, valve!!!!
